### ISSUE TITLE
Questions about the original dataset and fine-tuned checkpoints.

### ISSUE THREAD

**Issue by YuboFeng2023** [zhufq00/mcnc#4]
Hi Fanqi,

I hope this message finds you well.
I am highly interested in your research! 
Therefore, I kindly request you to provide me with the following mentioned in the paper:

1. the original dataset, which consist of 1,440,295 train examples, 10,000 dev examples and 10,000 test examples;
2. Fine-tuned model checkpoints, achieving a performance of 62.94 on the public datasets (refer to table 2);
3. Fine-tuned model checkpoints, achieving a performance of 67.21 on the original datasets (refer to table 4);

Should I have any future publications using your checkpoints, datasets, or related work, I will undoubtedly cite your paper!
My e-mail address is **argmax@126.com**

Best wishes in your endeavors!

**Comment by zhufq00** [comment]
Certainly, considering the demand for the original dataset by many, I have updated the download link in the readme. I previously attempted to share the 62.94 checkpoints, but unfortunately, they were corrupted, possibly due to an incomplete download initially. My suggestion is: by carefully selecting the checkpoint from the pretraining phase, I believe you will obtain similar results during replication. If the results you replicate deviate significantly from the reported values in the paper (for example, the accuracy drops by more than 1-2%), I will continue to assist you in your replication process.

I do not have the 67.21 checkpoints available for preservation, and due to significant changes in my machine environment, I am temporarily unable to provide them. Likewise, I am more than happy to offer any assistance to facilitate your replication process.

**Comment by YuboFeng2023** [comment]
Thank you for sharing! 
Your citations will be greatly increased.
